agent_service.go

```diff
@@ -0,0 +1,447 @@
+package godo
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+const agentConnectBasePath = "/v2/gen-ai/agents"
+
+type AgentService interface {
+	List(context.Context, *ListOptions) ([]*Agents, *Response, error)
+	Create(context.Context, *AgentCreateRequest) (*Agent, *Response, error)
+	Get(context.Context, string) (*Agent, *Response, error)
+	Update(context.Context, string, *AgentUpdateRequest) (*Agent, *Response, error)
+	Delete(context.Context, string) (*Agent, *Response, error)
+	UpdateVisibility(context.Context, string, *AgentVisibilityUpdateRequest) (*Agent, *Response, error)
+}
+
+var _ AgentService = &AgentServiceOp{}
+
+type AgentServiceOp struct {
+	client *Client
+}
+
+type genAIAgentsRoot struct {
+	Agents []*Agents `json:"agents"`
+	Links  *Links    `json:"links"`
+	Meta   *Meta     `json:"meta"`
+}
+
+type genAIAgentRoot struct {
+	Agent *Agent `json:"agent"`
+}
+
+type Agent struct {
+	AnthropicApiKey    *Info                `json:"anthropic_api_key,omitempty"`
+	ApiKeyInfos        []*Info              `json:"api_key_infos,omitempty"`
+	ApiKeys            []*ApiKeys           `json:"api_keys,omitempty"`
+	ChatBot            *ChatBot             `json:"chatbot,omitempty"`
+	ChatbotIdentifiers []ChatbotIdentifiers `json:"chatbot_identifiers,omitempty"`
```

Comment:
I think it's `agent_chatbot_identifier`
https://docs.digitalocean.com/reference/api/digitalocean/#tag/GenAI-Platform-(Public-Preview)/operation/genai_create_agent

under response.agent.chatbot_identifiers.agent_chatbot_identifier